testimony/cli.py

```diff
@@ -12,15 +12,19 @@
 @click.option('--tokens', help='Comma separated list of expected tokens')
 @click.option(
     '--minimum-tokens', help='Comma separated list of minimum expected tokens')
+@click.option(
+    '--token-configs', help='Yaml file with allowed values per token')
```

Comment:
Since it is a single file, maybe renaming to `token-config` here?

Reply:
Changed to just `config` in 94a49eda7be2a1362f939fe9127926f5e0f77170 (PR #142). Let's continue conversation in the other PR and close this one.